src/test/java/core/basesyntax/service/impl/ValidatorImplTest.java

```diff
@@ -0,0 +1,36 @@
+package core.basesyntax.service.impl;
+
+import static org.junit.Assert.assertTrue;
+
+import core.basesyntax.service.Validator;
+import org.junit.Test;
+
+public class ValidatorImplTest {
+    private static final Validator VALIDATOR = new ValidatorImpl();
+    private static final String FIRST_LETTERS_OF_TYPE_ACTIVITY = "bsrp";
+
+    @Test
+    public void validate_Ok() {
+        String[] typeActivity = FIRST_LETTERS_OF_TYPE_ACTIVITY.split("");
```

Comment:
Perhaps it's better to initially create an array of needed strings? Why do we need to use .split("") method?